frequency-chain/docs
# Dark mode svgs

Problem
=======
Add Dark themed SVGs and enable them when switching to Dark theme.  

Closes #74 

Solution
========

Change summary:
---------------
- [x] Add dark themed SVG images and alter/resize as needed
- [x] Remove white background from existing light themed SVGs
- [x] Import highlight CSS files only once
- [x] Apply the new SVG images to parsed doc output
- [x] Also ran `npm audit fix` to update some high vulnerabilities 

Steps to Verify:
----------------
```shell
git fetch
git checkout feat/dark-mode-svg
npm run serve
```
1. Go to each of Delegation, Token Allocation, and Relay Chain pages.
1. Switch from light to dark theme and back again.
1. make sure the images switch appropriately from dark to light
1. make sure they are the same size and the background fills are all in the same place. 
1. Disable JavaScript in the dev panel and verify that only the light themed SVGs appear.

Screenshots (optional):
-----------------------
![Oct-22-2024 12-57-27](https://github.com/user-attachments/assets/7614971f-f3b2-4a4e-88ac-f8da2e4418a9)
